adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -323,12 +323,21 @@ case class FeatureRDD(rdd: RDD[Feature],
   }
 
   /**
+   * Returns all reference regions that overlap this feature.
```

Comment:
In a different PR, I suggest adding doc to the definition in `GenomicRDD`, removing doc in all extensions of that method, and using the `override` keyword.  Does that work for Scala like it does in Java (i.e. the `@Override` annotation)?